web/pages/edit/index.page.tsx

```diff
@@ -42,6 +42,9 @@ import {
 	SymbolUploadStore,
 } from '../../store';
 import React from 'react';
+import EngineeringSymbolEditor from '../../components/symbolEditor/EngineeringSymbolEditor';
+import { SymbolEditorEvent } from '../../components/symbolEditor/models/EditorEvent';
+import { EditorCommandMessage } from '../../components/symbolEditor/models/EditorCommand';
```

Comment:
Mb add some `index.ts` to make import more beautiful ;)

Reply:
Done